specification/adp/resource-manager/readme.python.md

```diff
@@ -22,3 +21,8 @@ output-folder: $(python-sdks-folder)/adp/azure-mgmt-adp/azure/mgmt/adp
 basic-setup-py: true
 output-folder: $(python-sdks-folder)/adp/azure-mgmt-adp
 ```
+
+``` yaml $(python) && $(track2)
+modelerfour:
+  lenient-model-deduplication: true
```

Comment:
It's not recommended to use that option, since this may create terrible names. You should not put the option, and if you have a name conflict, investigate to be sure if something is not wrong in your Swagger